scripts/deploy-gateway.js

```diff
@@ -10,27 +10,31 @@ const {
 } = require('ethers');
 
 const { execSync }  = require('child_process');
+const path = require('node:path');
+
+const {
+  printLog,
+  printObj,
+} = require('./logging');
 
 // these environment variables should be defined in an '.env' file
+const contractsPath = process.env.CONTRACTS_PATH;
```

Comment:
```suggestion
const contractsPath = process.env.CONTRACTS_PATH || './build';
```
so that it's optional

Reply:
done